### ISSUE TITLE
Question : Anybody tried to extend the video ?

### ISSUE THREAD

**Issue by x4080** [Lightricks/LTX-Video#32]
Hi, is it possible to just put the last frame of the video generated so that it can generate more frames but consistently ? So this will be using img2vid then, will the prompt still the same ?

Thanks

**Comment by x4080** [comment]
I tried it myself using last frame of the txt2vid frame, and put it into img2vid workflow, the 1st frame result is nice but the reset is like over saturated, I'll try again with different seed

**Comment by x4080** [comment]
Yes using different seed works, the new problem is later the video starts to "hallucinate", is there different for img2vid prompt ?

**Comment by Shecht-ltx** [comment]
@x4080 Given the performance improvements and quality advancements in LTXV this week, we are now able to generate high-quality 12-14 second clips in LTXS. Best of all, this can be achieved while keeping waiting times very competitive.

**Comment by x4080** [comment]
@Shecht-ltx Thanks

**Comment by MyCroniesSoft** [comment]
so many ai gen models in last 3 months, some have been available since 2023, while i give genmo the crown for the opensource part but 48gig just to load the model, omg thats insane thats 1 ada 6000, then the second for ease is pyramid flow, for complexity on model truncated i give ltx, and lasty the one i use ie none of them its ????

That is impressive, while i like midways video generator, but not for 200$ a month with credits.

some chinese ones are rising like ive never seen, definitely a threat.

**Comment by x4080** [comment]
@MyCroniesSoft Is the brightness/contrast of the video changes when extending the video ? I was trying comfy ui ltx and its more efficient because it splits between GPU/CPU memory and with STG getting better result too

**Comment by bitnom** [comment]
It becomes decoherent for me every time. The saturation, brightness, and colors start to go. Entities blur and it goes bad. I've been using the one on replicate.
